apple/Sources/Sargon/Extensions/Swiftified/Profile+Supporting+Types/References/RefBytes+Swiftified.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  
+//
+//  Created by Alexander Cyon on 2024-05-01.
+//
+
+import Foundation
+import SargonUniFFI
+
+extension BagOfBytes: SargonModel {
+	public static let sample: Self = newBagOfBytesSampleAced()
```

Comment:
Can be moved elsewhere, but meh, actually just wanted it because I put constraint `Inner: SargonModel`. Remember that `BagOfBytes` **is** `Data`, but put `extension BagOfBytes` so it "echoes" with `newBagOfBytes` below.